rnote-ui/src/canvaswrapper.rs

```diff
@@ -167,8 +171,10 @@ mod imp {
             self.canvas.connect_notify_local(
                 Some("touch-drawing"),
                 clone!(@weak obj as canvaswrapper => move |canvas, _pspec| {
-                    // Disable the zoom gesture when touch drawing is enabled
+                    // enable the touch drag gesture & disable the zoom gesture and kinetic scrolling when touch drawing is enabled
+                    canvaswrapper.scroller().set_kinetic_scrolling(!canvas.touch_drawing() && canvaswrapper.inertial_scrolling());
                     canvaswrapper.canvas_zoom_gesture_enable(!canvas.touch_drawing());
+                    canvaswrapper.canvas_touch_drag_gesture_enable(canvas.touch_drawing());
```

Comment:
Okay, you're right, leaving it on when kinetic-scrolling is enabled doesn't do any harm. Explaining this deliberate decision in a comment here would be helpful for my forgetful mind

Reply:
> leaving it on when kinetic-scrolling is enabled doesn't do any harm

But kinetic-scrolling is only enabled if touch drawing is disabled, meaning that the gesture is only enabled if kinetic-scrolling is disabled. Or is that what you meant?

Reply:
I'll resolve, this works but disabling it at all might be unnecessary (see other comment)